v0/destinations/mautic/utils.js

```diff
@@ -0,0 +1,298 @@
+const { CustomError } = require("../../util");
+const { BASE_URL } = require("./config");
+const { httpGET } = require("../../../adapters/network");
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const _ = require("lodash");
+const { getFieldValueFromMessage } = require("../../util");
+const { set, values } = require("lodash");
+
+// Includes the fields that can be lookUpfields
+const FIELDS = [
+  "",
+  "title",
+  "firstName",
+  "lastName",
+  "role",
+  "email",
+  "phone",
+  "city",
+  "state",
+  "zipcode",
+  "country"
+];
+//All the titles that are allowed
+const ALLOWED_TITLES = ["Mr", "Mrs", "Miss", "Mr.", "Mrs.", "Miss."];
+
+//check if input is a date or not
+const isDate = date => {
+  return new Date(date) !== "Invalid Date" && !isNaN(new Date(date));
+};
+
+//Includes valid values for prospectOrCustomer
+const ALLOWED_POC = ["Prospect", "Customer"];
+
+//Includes valid Values for subscriptionStatus
+const ALLOWED_SUBSCRIPTION_STATUS = ["New", "Existing"];
+
+//Includes valid Values for role
+const ALLOWED_ROLE_VALUES = [
+  "Individual Contributor",
+  "Manager",
+  "Director",
+  "Executive",
+  "Consultant"
+];
+
+// to get the Mautic field for the given Rudder Field for Filter axios call
+const fieldMap={"title":"title",
```

Comment:
Use a better variable name and add better commenting, otherwise it seems confusing.

Reply:
I have changed it lookupFieldMap.